addtional_test.go

```diff
@@ -0,0 +1,88 @@
+package flagsfiller_test
+
+import (
+	"flag"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/itzg/go-flagsfiller"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTime(t *testing.T) {
+	type Config struct {
+		T time.Time
+	}
+
+	var config Config
+
+	filler := flagsfiller.New()
+
+	var flagset flag.FlagSet
+	err := filler.Fill(&flagset, &config)
+	require.NoError(t, err)
+
+	err = flagset.Parse([]string{"-t", "2001-03-02 13:04:21"})
+	require.NoError(t, err)
+	expeted, _ := time.Parse(time.DateTime, "2001-03-02 13:04:21")
```

Comment:
You can bump the Go version for the build

https://github.com/itzg/go-flagsfiller/blob/b3fbd381cf9e74fc96024784b43b3cac2336771d/.github/workflows/test.yml#L19

and module needs to be sync'ed up anyway

https://github.com/itzg/go-flagsfiller/blob/b3fbd381cf9e74fc96024784b43b3cac2336771d/go.mod#L3